server/readModel/lists/ownCaches.js

```diff
@@ -1,72 +1,81 @@
 'use strict';
 
 const fields = {
-	name: { initialState: '' },
-	description: {initialState: ''},
-	coordinate: {initialState: {lat: 0, lon: 0}},
-	published: {initialState: false},
-	countFavorites: {initialState: 0},
-	countFindings: {initialState: 0},
-	comments: {initialState: []}
+  name: { initialState: '' },
+  description: { initialState: '' },
+  coordinate: { initialState: { lat: 0, lon: 0 }},
+  published: { initialState: false },
+  countFavorites: { initialState: 0 },
+  countFindings: { initialState: 0 },
+  comments: { initialState: []}
 };
 
 const when = {
-	'geocaching.cache.hidden': (caches, event, mark) => {
-		caches.add({
-			name: event.data.name,
-			description: event.data.description,
-			coordinate: event.data.coordinate
-		});
-		mark.asDone();
-	},
+  'geocaching.cache.hidden': (ownCaches, event, mark) => {
```

Comment:
Is there a special reason why you use arrow functions here?

IMHO (but that's a matter of personal taste), using

`'geocaching.cache.hidden' (ownCaches, event, mark) {`

is more readable, as it contains less visual noise. But as said, that's up to you.

Reply:
No, there isn't a special reason. You are right, the readability is much better so I'll change it 👍